src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md

```diff
@@ -0,0 +1,257 @@
+{% endAside %}
+
+## Demo
+
+You can see the [QR code scanner demo](https://mediastreamtrack.glitch.me/) from the section above
+in action on a desktop or mobile browser. Hold a QR code in front of the camera and the app will
+detect it and highlight it. You can see the application's source code
+[on Glitch](https://glitch.com/edit/#!/mediastreamtrack?path=index.html%3A21%3A50).
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/VwysZHgnzswePs684xOJ.png", alt="QR code scanner running in desktop browser tab showing a detected and highlighted QR code on the phone the user holds in front of the laptop's camera.", width="800", height="481" %}
+
+## Security and Privacy considerations
+
+The security of this API relies on existing mechanisms in the web platform. As data is exposed using
+the `VideoFrame` and `AudioFrame` interfaces, the rules of those interfaces to deal with
+origin-tainted data apply. For example, data from cross-origin resources cannot be accessed due to
+existing restrictions on accessing such resources (e.g., it is not possible to access the pixels of a
+cross-origin image or video element). In addition, access to media data from cameras,
+microphones, or screens is subject to user authorization. The media data this API exposes is
+already available through other APIs. In addition to the media data, this API exposes some control
+signals such as requests for new frames. These signals are intended as hints and do not pose a
+significant security risk.
+
+## Feedback
+
+The Chromium team wants to hear about your experiences with insertable streams for
+`MediaStreamTrack`.
+
+### Tell us about the API design
+
+Is there something about the API that does not work like you expected? Or are there missing methods
+or properties that you need to implement your idea? Do you have a question or comment on the security
+model? File a spec issue on the corresponding [GitHub repo][github], or add your thoughts to an
+existing issue.
+
+### Report a problem with the implementation
+
+Did you find a bug with Chromium's implementation? Or is the implementation different from the spec?
+File a bug at [new.crbug.com](https://new.crbug.com). Be sure to include as much detail as you can,
+simple instructions for reproducing, and enter `Blink>MediaStream` in the **Components** box.
+[Glitch](https://glitch.com/) works great for sharing quick and easy repros.
+
+### Show support for the API
+
+Are you planning to use insertable streams for `MediaStreamTrack`? Your public support helps the
+Chromium team prioritize features and shows other browser vendors how critical it is to support
+them.
+
+Send a tweet to [@ChromiumDev][cr-dev-twitter] using the hashtag
+[`#InsertableStreams`](https://twitter.com/search?q=%23InsertableStreams&src=recent_search_click&f=live)
+and let us know where and how you are using it.
+
+## Helpful links
+
+- [Spec draft][spec]
+- [Explainer][explainer]
+- [ChromeStatus](https://chromestatus.com/features/5499415634640896)
```

Comment:
I think this link has a typo: s/features/feature/

Reply:
Thanks!